client/src/app/core/repositories/event-management/organisation-repository.service.ts

```diff
@@ -44,4 +46,12 @@ export class OrganisationRepositoryService extends BaseRepository<ViewOrganisati
             settings: settingsFieldset
         };
     }
+
+    public update(update: OrganizationAction.UpdatePayload): Promise<void> {
+        /**
+         * I suppose the orga id is always 1
+         */
+        update.id = 1;
+        return this.sendActionToBackend(OrganizationAction.UPDATE, update);
+    }
```

Comment:
For the moment okay, but we have ensure, that only values specified in `OrganizationAction.UpdatePayload` are sent to the backend.
Relates to #124

Reply:
ok